### PR TITLE
clarifications in seeking-consensus-2024-01.md

### PR DESCRIPTION
**Pull request by TallTed**
too many to list... made much `code` into non-code; tweaked line breaks; added bullets to help clarify breaks; adjusted whitespace

### CODE REVIEW

**Comment by afs** [comment]
Added commit with HTML, no other changes.
```
pandoc seeking-consensus-2024-01.md -f gfm -o seeking-consensus-2024-01.html \
       --metadata title="Seeking Consensus" -s -c seeking-consensus-2024-01.css
```

File: docs/seeking-consensus-2024-01.html [hunk 1]
```diff
@@ -141,6 +144,25 @@ <h1 id="seeking-consensus">Seeking consensus</h1>
...
+<li><pre><code>:e rdf:subject   :s .
+:e rdf:predicate :p .
+:e rdf:object    :o .
+:e :pp           :oo .
+## possibly also
+## :e       rdf:type       rdf:Statement .</code></pre></li>
+<li><pre><code>:e       rdf:nameOf _blank1
+_blank1  rdf:subject   :s .
+_blank1  rdf:predicate :p .
+_blank1  rdf:object    :o .
```
Comment (afs) [inline docs/seeking-consensus-2024-01.html:159]: _blank1 => _:blank1
```suggestion
<li><pre><code>:e       rdf:nameOf _:blank1
_:blank1  rdf:subject   :s .
_:blank1  rdf:predicate :p .
_:blank1  rdf:object    :o .
```

Comment (afs) [reply docs/seeking-consensus-2024-01.html:159]: (not in the HTML generated)

**Comment by afs** [comment]
> In fact, with this new row, I think we can even remove the row with the `COUNT(*)` query because it does not actually add anything anymore.

Agreed.

File: docs/seeking-consensus-2024-01.md [hunk 2]
```diff
@@ -68,6 +68,31 @@ pandoc seeking-consensus-2024-01.md -f gfm -o seeking-consensus-2024-01.html -s
...
+  :e :pp           :oo .
+  ## possibly also
+  ## :e       rdf:type       rdf:Statement .
+  ```
+- ```
+  :e       rdf:nameOf _blank1
+  _blank1  rdf:subject   :s .
+  _blank1  rdf:predicate :p .
+  _blank1  rdf:object    :o .
+  :e       :pp           :oo .
```
Comment (TallTed) [inline docs/seeking-consensus-2024-01.md:88]:
```suggestion
  :e       rdf:nameOf _:blank1
  _:blank1  rdf:subject   :s .
  _:blank1  rdf:predicate :p .
  _:blank1  rdf:object    :o .
  :e       :pp           :oo .
```

**Comment by TallTed** [comment]
I think the `COUNT(*)` row doesn't subtract from the rest of the table, and may help others who look at it later, given that the new row remains. 

That is, the `COUNT(*)` doesn't hurt and may help... so I'm leaving it. Someone else can make another PR to remove it, if they feel strongly about doing so.

**Review by afs — APPROVED** [review]

**Review by hartig — APPROVED** [review]

**Pull request merged**